pkg/codegen/go/gen.go

```diff
@@ -3293,6 +3388,7 @@ func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error
 
 			importsAndAliases := map[string]string{}
 			pkg.getImports(r, importsAndAliases)
+			importsAndAliases["github.com/pulumi/pulumi/sdk/v3/go/pulumi"] = ""
```

Comment:
Can you explain why this has moved? I would expect _functions_ to use it at least. (It seems like I'm wrong though.)

Reply:
Yes: enums do not require the `pulumi` SDK if they do not have input or output versions.